src/Compilers/CSharp/Test/Emit/Microsoft.CodeAnalysis.CSharp.Emit.UnitTests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp.UnitTests</RootNamespace>
-    <TargetFrameworks>net7.0;net472</TargetFrameworks>
+    <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
It looks like some other projects still reference TFM net7.0 as well. For example `EndToEnd` and `IOperations` unittests. Should they be updated too? I don't know if it's intentional. May be good to check with Jared.

Reply:
@jjonescz @jaredpar Should we update all tests to use `net8.0` TFM, instead of piecemeal?

Reply:
Seems reasonable at this point

Reply:
https://github.com/dotnet/roslyn/pull/69669